modules/osd-aws-privatelink-firewall-prerequisites.adoc

```diff
@@ -1,77 +1,118 @@
 
-See link:https://access.redhat.com/documentation/en-us/openshift_container_platform/4.7/html/installing/installation-configuration#configuring-firewall_configuring-firewall[Configuring your firewall for OpenShift Container Platform] for information about granting access to sites that OCP requires.
-
 .Procedure
 
 . Allowlist the following URLs that are used to install and download packages and tools:
 +
-[cols="2,4,2",options="header"]
+[cols="3,4,1",options="header"]
 |===
-|URL | Function | Requirement
+|URL | Function | Port
 |`registry.redhat.io`
-|Provides core components such as dev tools and operator based add-ons, and Red Hat provided container images including middleware and Universal Base Image.
-|Required
+|Required. Provides core components such as dev tools and operator based add-ons, and Red Hat provided container images including middleware and Universal Base Image.
+|443, 80
 
 |`quay.io`
-|Used by the cluster to download the platform container images
-|Required
+|Required. Used by the cluster to download the platform container images.
+|443, 80
+
+|`.quay.io`
+|Required. Used by the cluster to download the platform container images.
+|443, 80
 
 |`sso.redhat.com`
-|The `https://cloud.redhat.com/openshift` site uses authentication from `sso.redhat.com` to  download the pull secret and use Red Hat SaaS solutions to facilitate monitoring of your subscriptions, cluster inventory, chargeback reporting, and so on.
-|Required
+|Required. The `https://cloud.redhat.com/openshift` site uses authentication from `sso.redhat.com` to  download the pull secret and use Red Hat SaaS solutions to facilitate monitoring of your subscriptions, cluster inventory, chargeback reporting, and so on.
+|443, 80
 
 |`pull.q1w2.quay.rhcloud.com`
-|Provides a fallback registry used by the cluster when quay.io is not available.
-|Recommended
+|Recommended. Provides a fallback registry used by the cluster when quay.io is not available.
+|443, 80
+
+|`*.q1w2.quay.rhcloud.com`
+|Recommended. Enables a CNAME to resolve to a regionalised endpoint.
+|443, 80
+
+|`openshift.org`
+|Required. Provides Red Hat Enterprise Linux CoreOS (RHCOS) images.
+|443, 80
+
+|`console.redhat.com`
+|Required. Allows interactions between the cluster and OpenShift Console Manager (OCM) to enable functionality, such as scheduling upgrades.
+|443, 80
+
+|`quay-registry.s3.amazonaws.com`
+|Required. Used to access Quay image content in AWS.
+|443, 80
 |===
 +
 When you add a site such as `quay.io` to your allowlist, do not add a wildcard entry such as `*.quay.io` to your denylist. In most cases, image registries use a content delivery network (CDN) to serve images. If a firewall blocks access, then image downloads are denied when the initial download request is redirected to a host name such as `cdn01.quay.io`.
++
+CDN host names, such as `cdn01.quay.io`, are covered when you add a wildcard entry, such as ``*.quay.io`, in your allowlist.
```

Comment:
```suggestion
CDN host names, such as `cdn01.quay.io`, are covered when you add a wildcard entry, such as ``.quay.io`, in your allowlist.
```

Given this doc is strictly AWS specific, lets be consistent with the expectations of how to configure a wildcard?